llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1832,6 +1864,39 @@ SmallVector<int64_t> PackOp::getStaticTiles() {
   return ::getStaticTiles(*this);
 }
 
+SmallVector<OpFoldResult> PackOp::getResultShape(
```

Comment:
Do we need the method separately? Can we move the implementation to `PackOp::getResultShape(OpBuilder &builder)`?

Reply:
See comment above. This is a static method to help creating the `pack` op (i.e getting the result shape without needing an instance of the `pack` op itself)